edu-blog-vue/components/AuthorBio.vue

```diff
@@ -1,9 +1,9 @@
 <template>
   <aside>
-    <div class="bio">
-      <img class="bio__img" :src="image" :alt="alt" />
+    <div class="author-bio">
+      <img class="author-bio__img" :src="image" :alt="alt" />
 
-      <div class="bio__text">
+      <div class="author-bio__text">
```

Comment:
you have no styling for that class right now. probably you can recycle the class on the p-tag (see comment below)